db/people.yml

```diff
@@ -1993,10 +2013,17 @@ jwilley:
       group: bg
       advisor: sbillinge
       status: masters
+    - begin_date: 2019-09-16
```

Comment:
please could you add this,  which helps me keep track of who is in the group when:
```
    - begin_date: 2021-05-20
      position: masters researcher
      organization: columbiau
      advisor: sbillinge
      group: bg
```